source/app/common.h

```diff
@@ -250,6 +251,13 @@ struct titleEntry {
     std::optional<filePart> customFile;
 };
 
+struct dumpFileFilter {
+    std::vector<std::string> fileNames;
+    std::vector<std::string> extensions;
+
+    std::vector<std::string> outMatchedFiles;
+};
+
```

Comment:
This is required since i need to get a list of the file paths that ends with "task.db", this avoids having to implement more useless recursive code, benefiting from the existing dump code